audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/config/AuditConfig.java

```diff
@@ -15,6 +22,12 @@
     @Value("${altus.audit.port:8989}")
     private int port;
 
+    @Inject
+    private List<EventDataUtilizer> eventDataUtilizerImplementations;
```

Comment:
after renaming the template class, the field name would be like `eventDataConverters`

Reply:
still, no conversion happens there

Reply:
AuditEventDataUpdaters ? as suggested above. Thoughts?